modules/installation-gcp-config-yaml.adoc

```diff
@@ -17,6 +17,11 @@ ifeval::["{context}" == "installing-gcp-private"]
 :private:
 :vpc:
 endif::[]
```

Comment:
The replica of work nodes should be 0

Reply:
Hi @jstuever , please help confirm: in shared VPC scenario, only support to create worker nodes manually, correct? 
In UPI installation of standalone project, cluster could provision the worker nodes, does it work for shared VPC? Actually I tried that but worker nodes were not created. I am not sure if it's expected.